dask/dataframe/shuffle.py

```diff
@@ -329,11 +330,22 @@ def __reduce__(self):
 
     def __call__(self, *args, **kwargs):
         import partd
-
+        # Try to load partd compression module. Silently fail, if compression algorithm is not found.
+        try:
+            partd_compression = (
+                getattr(partd.compressed, self.compression)
+                if self.compression
+                else None
+            )
+        except AttributeError:
+            raise AttributeError('Not able to import and load {0} as compression algorithm. Is it supported by Partd or installed?')
```

Comment:
I think you're missing a `.format`. Can you add a test that hits this?

You shouldn't need the `0` in the placeholder.

Reply:
This exception should likely be a ValueError, not an AttributeError.